app/routes/accounts.$id.tsx

```diff
@@ -129,3 +158,28 @@ const AccountTimeline = ({ notes }: AccountTimelineProps) => {
     </div>
   );
 };
+
+const EmptyAccountTimeline = ({
+  accountName,
+  isThisAccountSelf,
+}: {
+  accountName: string;
+  isThisAccountSelf: boolean;
+}) => {
+  return (
+    <div className={styles.emptyState}>
+      <span>💭</span>
+      {isThisAccountSelf ? (
+        <>
+          <h3>No notes yet</h3>
+          <p>Your notes will appear here when you post them.</p>
+        </>
+      ) : (
+        <>
+          <h3>No notes yet</h3>
+          <p>{accountName} hasn&#39;t made any notes yet.</p>
+        </>
+      )}
```

Comment:
```suggestion
      <h3>No notes yet</h3>
      {isThisAccountSelf ? (
        <p>Your notes will appear here when you post them.</p>
      ) : (
        <p>{accountName} hasn&#39;t made any notes yet.</p>
      )}
```
どういう意図かわかりませんが, `h3` を外に出して共通化したほうがよいかと思います.